index.js

```diff
@@ -39,29 +44,50 @@ const login = async (page, url, id, pw) => {
   });
   const signInSelector = '.row-fluid:nth-child(3) button:nth-child(1)';
   const signOutSelector = '.row-fluid:nth-child(3) button:nth-child(2)';
-  const signInSystemSelector = '.row-fluid:nth-child(6) button';
-  const lookupSelector = '.row-fluid:nth-child(5) button';
+  const lookupSelector = '.row-fluid:nth-child(4) button:nth-child(1)';
   // 學校日曆
   const calendarUrl = 'https://eadm.ncku.edu.tw/welldoc/iftwf/WF8F11A.php?f_menuname=%AD%D3%A4H%AEt%B0%B2%A4%EB%BE%E4%AA%ED'
   // 登入行政 e 化系統
-  const eSystemSelector = '.row-fluid:nth-child(6) button';
+  const eSystemSelector = '.row-fluid:nth-child(5) button';
   const action = program.action;
 
+  console.log(`${process.argv[1]}`)
   // 打卡 or 登入
   await login(page, signInUrl, credential.id, credential.password);
-  // 取得放假日
-  await page.click(eSystemSelector);
-  // 一定要等久一點，不然會非法登入
-  await delay(3000);
-  await page.goto(calendarUrl);
-  // 抓取 紅色日子
-  await delay(3000);
-  const text = await page.evaluate(() => Array.from(document.querySelectorAll('table:nth-child(2) :nth-child(1) td[bgcolor="#FFCCCC"]'), element => element.textContent));
-  const holidays = text.map(Number);
+
+
+  var today = new Date(), y = today.getFullYear(), m = today.getMonth();
+  var lastDay = new Date(y, m + 1, 0);
+
+  // 如果今天是這個月的最後一天，把下個月的假期存起來
+  if(today.getTime() == lastDay.getTime()) {
```

Comment:
Did you test this? It seems that you compare to "time".

Should it be like this?
https://stackoverflow.com/questions/6355063/how-to-validate-date-if-is-the-last-day-of-the-month-with-javascript

Reply:
I think I should change `==` operator `===`. What do you think ?

[Get Last Day](https://stackoverflow.com/questions/13571700/get-first-and-last-date-of-current-month-with-javascript-or-jquery/13572682)

[Determine the Date are equal](https://stackoverflow.com/questions/492994/compare-two-dates-with-javascript/497790)

Reply:
In JS, Date object just like Python's datetime includes time so I think this won't work.

for example:
```
> new Date()
< Fri Aug 21 2020 13:26:31 GMT+0800 (CST)
> new Date('2020', 9, 0)
< Wed Sep 30 2020 00:00:00 GMT+0800 (CST)
```

Reply:
The correct way might be like this

```js
function isLastDay(dt) {
    var test = new Date(dt.getTime()),
        month = test.getMonth();

    test.setDate(test.getDate() + 1);
    return test.getMonth() !== month;
}
```

Reply:
Yes, you are right. I have tested it and fix it in 205907c93ac77d0ed20055c88db2138bc03aa32e. Thanks a lot ~